<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockCharmap</class>
 <widget class="QDockWidget" name="DockCharmap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Selector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QToolBox" name="toolBoxFilters">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="pageCategories">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>132</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Categories</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListView" name="listViewCategories">
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageScripts">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>132</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Scripts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="listViewScripts">
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageDecompositions">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>132</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Decompositions</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListView" name="listViewDecompositions">
           <property name="selectionMode">
            <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageMSB">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>132</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Values</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBoxValueFilter">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelMSB">
              <property name="text">
               <string>MSB</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelUnicodeFrom">
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelUnicodeTo">
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="spinBoxMSB">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxUnicodeFrom">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="spinBoxUnicodeTo">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>75</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageText">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>310</width>
          <height>132</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Text</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEditFilter"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableView" name="tableViewCharmap"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBoxFilters">
      <property name="title">
       <string>Filters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonClearFilters">
      <property name="text">
       <string>Clear All Filters</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
